### DISCUSSION TITLE
Add links for `table_name_sysid` format field values

### DISCUSSION THREAD

**Discussion by VanWeapon** [arnoudkooi/SN-Utils#447] [Ideas]
This formatting exists in a few places in the platform but it's kinda awkward to navigate to the record when it's presented in this format, 
e.g. in the Upgrade Detail record: 
![image](https://github.com/arnoudkooi/ServiceNow-Utils/assets/30616047/f239c887-7bcf-4963-b721-75359c3f8760)

It would be good if SNU could do the following
- Fields with values in this format have a link next to them to open the record
- Lists with these values have that value turned into a hyperlink 
- Slashcommands could recognise a pasted value in this format, same as it recognises plain sys_ids, and navigates to that record

**Comment by VanWeapon** [comment]
just noticed the list one is there already for update sets, nice

**Comment by arnoudkooi** [comment]
> * Fields with values in this format have a link next to them to open the record]

Won't implement this one.
> * Lists with these values have that value turned into a hyperlink

I added the table sys_upgrade_history_log, be sure to not have the filename column as the first, then it will not work.
> * Slashcommands could recognize a pasted value in this format, the same as it recognizes plain sys_ids, and navigates to that record

I'll add this in 7.1.6.0
